### ISSUE TITLE
bump the peer dependency to support react v18

### ISSUE THREAD

**Issue by weidonglian** [kqito/use-tus#35]
```
  "peerDependencies": {
    "react": "^16.8.0 || ^17.0.0 || ^18.0.0",
    "tus-js-client": "^2.2.0"
  },
```
we need to add `|| ^18.0.0`. 

As of today now, we have to force install via `npm install --force`.

**Comment by kqito** [comment]
I'm sorry for the delay.

This bug has been fixed in v7.0, since the version was changed to `">=16.8"`!
Thank you for your report !

https://github.com/kqito/use-tus/releases/tag/v0.7.0

**Comment by weidonglian** [comment]
Thanks so much:).
